crates/viewer/re_viewer/src/ui/top_panel.rs

```diff
@@ -426,103 +448,75 @@ fn memory_use_label_ui(ui: &mut egui::Ui, gpu_resource_stats: &WgpuResourcePoolS
     }
 }
 
-fn latency_ui(ui: &mut egui::Ui, app: &mut App, store_context: Option<&StoreContext<'_>>) {
-    if let Some(response) = e2e_latency_ui(ui, store_context) {
-        // Show queue latency on hover, as that is part of this.
-        // For instance, if the framerate is really bad we have less time to ingest incoming data,
-        // leading to an ever-increasing input queue.
-        let rx = app.msg_receive_set();
-        let queue_len = rx.queue_len();
-        let latency_sec = rx.latency_nanos() as f32 / 1e9;
-        // empty queue == unreliable latency
-        if 0 < queue_len {
-            response.on_hover_ui(|ui| {
-                ui.label(format!(
-                    "Queue latency: {}, length: {}",
-                    latency_text(latency_sec),
-                    format_uint(queue_len),
-                ));
-
-                ui.label(
-                    "When more data is arriving over network than the Rerun Viewer can ingest, a queue starts building up, leading to latency and increased RAM use.\n\
-                         We call this the queue latency.");
-            });
-        }
```

Comment:
I removed the old "queue latency", which was just measuring the latency from receiving data until ingesting it; which we now cover in another way with the new code